Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/SimpleStepEntry.kt

```diff
@@ -25,6 +26,8 @@ class SimpleStepEntry @JvmOverloads constructor(
 
             simple_step_entry_body.text =
                 getText(R.styleable.SimpleStepEntry_simple_step_entry_text)
+
+            if (simple_step_entry_body.text.isEmpty()) simple_step_entry_body.visibility = View.GONE
```

Comment:
Can we change this to "findViewById" or ViewBinding in this PR? Kotlin synthetics will be removed in a future Android update.

https://developer.android.com/topic/libraries/view-binding/migration